bigframes/pandas/__init__.py

```diff
@@ -695,6 +695,48 @@ def to_datetime(
 to_datetime.__doc__ = vendored_pandas_datetimes.to_datetime.__doc__
 
 
+def get_default_session_id() -> str:
+    """Gets the default session id. This is the session id that is
+    used whenever a custom session has not been provided. It is the
+    session id of the default global session. It is prefixed to the
+    table id of all temporary tables created in the global session.
+
+    Returns:
+        str, the default global session id, ex. 'sessiona1b2c'
+    """
+    return get_global_session().session_id
+
+
+def clean_up_by_session_id(session_id: str) -> None:
+    """Searches through datasets and table names in Bigquery and
```

Comment:
I don't think we need to loop through datasets. There is only one dataset per user + project + location combination that the temp tables would be in and that's the one we discover here:

https://github.com/googleapis/python-bigquery-dataframes/blob/0e24036fe33c60cef08ba78e7ce69319338b6b03/bigframes/session/__init__.py#L279-L288

Likely we'll want to refactor that function so we can re-use it here. Maybe put a function in https://github.com/googleapis/python-bigquery-dataframes/blob/main/bigframes/session/_io/bigquery.py that takes a bqclient, location, and project and returns the anonymous dataset?